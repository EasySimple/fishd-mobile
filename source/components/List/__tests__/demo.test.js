import webDemoTest from '../../../tests/demoTest';

webDemoTest('List');
