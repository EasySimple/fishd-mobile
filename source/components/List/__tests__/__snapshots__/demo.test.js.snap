// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders List demo correctly 1`] = `
<div>
  <div>
    <div
      class="am-list my-list"
    >
      <div
        class="am-list-header"
      >
        Basic Style
      </div>
      <div
        class="am-list-body"
      >
        <div
          class="am-list-item am-list-item-middle"
        >
          <div
            class="am-list-line"
          >
            <div
              class="am-list-content"
            >
              Title
            </div>
            <div
              class="am-list-extra"
            >
              extra content
            </div>
          </div>
          <div
            class="am-list-ripple"
            style="display:none"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
